content/geometry/file-format-support.adoc

```diff
@@ -1,9 +1,34 @@
 = File format
 
-ASAM OpenMATERIAL does not define its own 3D model format. ASAM OpenMATERIAL defines coordinate systems, units, node hierarchies, and more, which can be applied in different 3D model formats.
+ifndef::include-only-once[]
+:root-path: ../
+:imagesdir: {root-path}/_images
+:includedir: {root-path}/_images
+include::{root-path}_config.adoc[]
+endif::[]
 
-ASAM OpenMATERIAL has tested and supports the following file formats for 3D models:
+ASAM OpenMATERIAL does not define its own 3D model format.
+The geometry specification however defines coordinate systems, units, node hierarchies, and more, which can be applied in different 3D model formats.
 
-* glTF
-* FBX
-* USD
+ASAM OpenMATERIAL supports the 3D model file formats glTF, FBX and USD with the following considerations.
+
+**glTF**
+
+ASAM OpenMATERIAL 3D models in glTF format SHALL conform to the official https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html[glTF specification].
+
+The glTF coordinate system is defined as a right-handed system with +Y as up, +Z as forward, and -X as right. The front of a glTF asset faces +Z.
+This differs from how coordinate systems are defined in OpenMATERIAL, see xref:general.adoc[].
```

Comment:
@FKlopfer could you add a reference here to the "Coordinate systems" section of the general file? I was not sure how to reference a certain section within a file.

Reply:
To reference a subsection, you have to add the automatically created anchor to the link. In this case, it is like this: "xref:geometry/general.adoc#_coordinate_systems[Coordinate sytems]"